include/mptcpd/path_manager.h

```diff
@@ -144,13 +141,124 @@ MPTCPD_API int mptcpd_pm_remove_addr(struct mptcpd_pm *pm,
+                      mptcpd_aid_t local_address_id,
+                      mptcpd_aid_t remote_address_id,
+                      struct sockaddr const *local_addr,
+                      struct sockaddr const *remote_addr,
+                      bool backup);
+
+/**
+ * @brief Set priority of a subflow.
+ *
+ * @param[in] pm                The mptcpd path manager object.
+ * @param[in] token             MPTCP connection token.
+ * @param[in] local_addr        MPTCP subflow local address
+ *                              information, including the port.
+ * @param[in] remote_addr       MPTCP subflow remote address
+ *                              information, including the port.
+ * @param[in] backup            Whether or not to set the MPTCP
+ *                              subflow backup priority flag.
+ *
+ * @return @c 0 if operation was successful. @c errno otherwise.
+ */
+MPTCPD_API int mptcpd_pm_set_backup(
+        struct mptcpd_pm *pm,
+        mptcpd_token_t token,
+        struct sockaddr const *local_addr,
+        struct sockaddr const *remote_addr,
+        bool backup);
+
+/**
+ * @brief Remove a subflow.
+ *
+ * @param[in] pm                The mptcpd path manager object.
+ * @param[in] token             MPTCP connection token.
+ * @param[in] local_addr        MPTCP subflow local address
+ *                              information, including the port.
+ * @param[in] remote_addr       MPTCP subflow remote address
+ *                              information, including the port.
+ *
+ * @return @c 0 if operation was successful. @c errno otherwise.
+ */
+MPTCPD_API int mptcpd_pm_remove_subflow(
+        struct mptcpd_pm *pm,
+        mptcpd_token_t token,
+        struct sockaddr const *local_addr,
+        struct sockaddr const *remote_addr);
+///@}
+
 /**
  * @name Server-oriented Path Management Commands
  *
  * Server-oriented path management commands supported by the upstream
- * Linux kernel.  Path management is handled by the kernel.
+ * Linux kernel.  Path management is handled by the in-kernel path
+ * manager.
  */
 ///@{
+/**
+ * @brief Advertise new network address to peers.
+ *
+ * @param[in] pm    The mptcpd path manager object.
+ * @param[in] addr  Local IP address and port to be advertised
+ *                  through the MPTCP protocol @c ADD_ADDR
+ *                  option.  The port is optional, and is
+ *                  ignored if it is zero.
+ * @param[in] id    MPTCP local address ID.
+ * @param[in] flags Bitset of MPTCP flags associated with the network
+ *                  address, e.g. @c MPTCP_PM_ADDR_FLAG_BACKUP @c |
+ *                   @c MPTCP_PM_ADDR_FLAG_SUBFLOW.  Optional for
+ *                  upstream kernel (e.g. set to zero).
+ * @param[in] index Network interface index.  Optional for upstream
+ *                  Linux kernel (e.g. set to zero).
+ *
+ * @todo Should we define corresponding mptcpd flags instead of
+ *       exposing the flags in <linux/mptcp.h>?
```

Comment:
I do think it would be best to use our own flags - looks like we discussed in #83 and decided to not expose the kernel flags.

Reply:
This looks like a stale TODO comment.  The mptcpd-specific flags were added to [<mptcpd/types.h>](https://github.com/ossama-othman/mptcpd/blob/ff95363709aa6e725f6f8346bef1cfcb1d095958/include/mptcpd/types.h#L53-L60) in commit #116.  Documentation for the flags was subsequently added in #126.

I'll remove the stale comment.

Reply:
Fixed in commit a3bd6f7 in this pull request.